packages/subgraph/tasks/deploy.sh

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env bash
+
+set -eux
+
+JQ="npx --package=node-jq -- jq"
```

Comment:
A note here: should start to rely `jq` declared in flake.nix going forward. for local development without using Nix, install it yourself.

Reply:
that's achieved by simply using `jq` here, right? Letting the PATH sort it out.

Reply:
Yea, but it was added to node_modules, because developers were reporting the jq related errors.

No need to do it here ad-hoc, I will think about some fool-proof way of notifying user about they don't have dependencies installed such as jq.